build-gradle-plugin/build.gradle

```diff
@@ -1,30 +1,23 @@
 plugins {
     id("org.gradle.kotlin.kotlin-dsl") version "1.4.2"
```

Comment:
kotlin-dsl can be removed i think

Reply:
i cant :( whole plugin is written in Kotlin